holoviews/plotting/bokeh/raster.py

```diff
@@ -56,28 +46,12 @@ def _glyph_properties(self, plot, element, source, ranges):
         properties = super(RasterPlot, self)._glyph_properties(plot, element,
                                                                source, ranges)
         properties = {k: v for k, v in properties.items()}
-        val_dim = [d.name for d in element.vdims][0]
-        low, high = ranges.get(val_dim)
-        if 'cmap' in properties:
-            palette = mplcmap_to_palette(properties.pop('cmap', None))
-        colormapper = LogColorMapper if self.logz else LinearColorMapper
-        cmap = colormapper(palette, low=low, high=high)
-        properties['color_mapper'] = cmap
-        if 'color_mapper' not in self.handles:
-            self.handles['color_mapper'] = cmap
+        val_dim = [d for d in element.vdims][0]
+        properties['color_mapper'] = self._get_colormapper(val_dim, element, ranges,
+                                                           properties)
         return properties
```

Comment:
I'm just wondering why in this case ``self._get_colormapper`` is used to add the color mapper to the properties (in ``_glyph_properties``) but is added to the data in ``get_data`` everywhere else...